h/presenters.py

```diff
@@ -33,13 +35,13 @@ def truncate(content, length=60):
     link_text = truncate(link_text)
 
     if href and hostname:
-        link = '<a href="{href}" title="{title}">{link_text}</a><br>({hostname})'
-    elif hostname:
-        link = '<a title="{title}">{link_text}</a><br>({hostname})'
+        link = '<a href="{href}" title="{title}">{link_text}</a><br>{hostname}'
     elif href:
         link = '<a href="{href}" title="{title}">{link_text}</a>'
     else:
-        link = '<a title="{title}">{link_text}</a>'
+        link = '<em>Local file:</em> {title}'
+        if hostname:
+            link += '<br>{hostname}'
```

Comment:
Here is some data of how many annotations we have where the `url` is not one of `http://`, `https://`, `file://`

```
 1.80% - urn:x-pdf:
 0.07% - blob:http://
 0.04% - zotero://
 0.02% - ftp://
 0.02% - [object ...]
 0.01% - chrome-extension://
<0.01% - blob:(some id)
<0.01% - urn:x_pdf:
```

Considering that most of these uris can be considered local files, I think we can leave it like this.